letsencrypt/client/auth_handler.py

```diff
@@ -203,7 +203,9 @@ def _assign_responses(self, flat_list, ichall_dict):
 
     def _path_satisfied(self, dom):
         """Returns whether a path has been completely satisfied."""
-        return all(self.responses[dom][i] is not None for i in self.paths[dom])
+        return all(
+            self.responses[dom][i] is not None and
+            self.responses[dom][i] is not False for i in self.paths[dom])
```

Comment:
yeah... I think self.responses[dom][i] is best.  I found the initial not False easier to reason about knowing the API.  will change... thanks :+1: